CONTRIBUTING.md

```diff
@@ -29,9 +29,7 @@ run the tests, it can be useful to prepare a
 [virtual environment](https://docs.python.org/3/tutorial/venv.html) and install
 certain libraries typeshed uses to check stub files.
 
-Follow platform-specific instructions below. Following that, to automatically
-check your code before committing, you can copy the file `pre-commit` to
-`.git/hooks/pre-commit`.
+Follow platform-specific instructions below.
```

Comment:
```suggestion
Note that this step is entirely optional, as your code will be
automatically reformatted when your PR is submitted
(see **Code Formatting** below),
and the full test suite will also be run automatically.

Follow platform-specific instructions below.
```

Reply:
I'll try to make a commit along those lines :)

Reply:
I absolutely agree that running the tests locally is valuable! But in the **Contribution process at a glance** section, it says "Optionally run the tests". The wording here sounds much stronger at the moment -- I think the two sections should be consistent with each other 🙂 Also, I've often found typeshed's CI to be _far_ faster than my machine (but, maybe that's just a sign I need a new laptop...)